src/video-grid/VideoTile.module.css

```diff
@@ -96,7 +96,7 @@ limitations under the License.
 
   display: flex;
   align-items: center;
-  justify-content: flex-end;
+  justify-content: space-between;
```

Comment:
This won't affect the layout of the toolbar when it holds multiple buttons, will it?

Reply:
No, it should not since they are wrapped in a container of their own